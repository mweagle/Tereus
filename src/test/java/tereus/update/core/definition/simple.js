/* global CloudFormationUpdate,Patch */
CloudFormationUpdate("SomePatch")({
  "Key":
  {
    "Subkey" : Patch.Add("Value")
  }
});