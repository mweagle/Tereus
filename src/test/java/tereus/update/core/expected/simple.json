[{"op":"add","path":"/Key/Subkey","value":"Value"}]